examples/wasm/envoy_filter_http_wasm_example.cc

```diff
@@ -22,6 +229,10 @@ class ExampleContext : public Context {
   void onDone() override;
   void onLog() override;
   void onDelete() override;
+
+private:
+  std::string content_type;
+  struct Config config;
```

Comment:
`config_` for class members, same for others.

Reply:
Updated in [this commit](https://github.com/dengyijia/envoy-wasm/pull/1/commits/b44fa56d662316f60294b4745363e7ea67139943).